app/services/idv/steps/send_link_step.rb

```diff
@@ -63,7 +63,10 @@ def update_document_capture_session_requested_at(session_uuid)
       end
 
       def link(session_uuid)
-        idv_capture_doc_dashes_url('document-capture-session': session_uuid)
+        idv_capture_doc_dashes_url(
+          'document-capture-session': session_uuid,
+          request_id: sp_session[:request_id],
```

Comment:
did we also try the approach of looking up the session via the `users.unique_session_id` key and the out-of-band access?

Reply:
I just followed the same thing that we are already using for sending the email confirmation link store_sp_metadata_in_session